content/rest/reference/gitignore.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Gitignore
+intro: 'The Gitignore API fethes all the available [.gitignore templates](https://github.com/github/gitignore) that can be used to ignore the dependencies from adding in your project's repositories.' 
```

Comment:
Since the intro includes an apostrophe, you need to use double quotes instead of single quotes. Also typo fix:
```suggestion
intro: "The Gitignore API fetches all the available [.gitignore templates](https://github.com/github/gitignore) that can be used to ignore the dependencies from adding in your project's repositories."
```